src/mix/api/entities.ts

```diff
@@ -22,6 +23,25 @@ import {MixClient, MixResponse} from '../types'
 
 const debug = makeDebug('mix:api:entities')
 
+/**
+ * Configure a new entity in a project.
+ *
+ * @category entities
+ */
+export async function configureEntity(client: MixClient, params: EntitiesConfigureParams): Promise<MixResponse> {
+  debug('configureEntity()')
+  const {entityName, projectId, ...bodyParams} = params
+  const body = buildCreateOrUpdateEntityBody(bodyParams)
```

Comment:
I'm not sure how I feel about a function with `or` in its name. Would it be messier/more confusing to abstract this method name to something more generic -- maybe with an underscore -- then have both `buildCreateEntityBody` and `buildUpdateEntityBody` return calls to it?

Reply:
As discussed, I introduced an internal `buildEntityPayload` function that gets called by the two you suggested.